src/Microsoft.AspNet.SignalR.Client.Store/Microsoft.AspNet.SignalR.Client.Store.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.AspNet.SignalR.Client</RootNamespace>
-    <AssemblyName>Microsoft.AspNet.SignalR.Client.Store</AssemblyName>
```

Comment:
Here's the original PR that added this project: https://github.com/SignalR/SignalR/pull/3028/

I don't see any discussion as to why we went with multiple dlls despite the downside of never automatically choosing the Websocket transport. A quick check of a few follow up PRs also came up empty.

I think making this a single dll is a good change.